highlighting/lib/languages/1c.dart

```diff
@@ -2,6 +2,7 @@
 
 import '../src/mode.dart';
 import '../src/common_modes.dart';
+import 'common/nulls.dart';
```

Comment:
To avoid 193 changed files when we change imports in modes, I suggest making a single file like `src/mode_imports.dart` and export these 3 files there. Then in all modes only import this single file.

This way we can also get rid of a condition when importing `common/callbacks.dart`.

Reply:
?